### PR TITLE
docs: Add links to Pacstall packages

### PR DESCRIPTION
**Pull request by ghost**
Zed packages were recently added to Pacstall, giving one more way to install zed for users of Ubuntu and derivatives.

https://pacstall.dev/packages/zed-editor-stable-bin
https://pacstall.dev/packages/zed-editor-preview-bin

Thank you for creating Zed! :pray:

### CODE REVIEW

File: docs/src/linux.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Zed is open source, and [you can install from source](./development/linux.md).
 There are several third-party Zed packages for various Linux distributions and package managers, sometimes under `zed-editor`. You may be able to install Zed using these packages:
 
 - Flathub: [`dev.zed.Zed`](https://flathub.org/apps/dev.zed.Zed)
+- Ubuntu (Pacstall): [`zed-editor-stable-bin`](https://pacstall.dev/packages/zed-editor-stable-bin) [`zed-editor-preview-bin`](https://pacstall.dev/packages/zed-editor-preview-bin)
```
Comment (maxdeviant) [inline docs/src/linux.md:39]: Can we move this entry down after "openSUSE Tumbleweed"?

**Comment by notpeter** [comment]
It looks like you are failing to preserve the `licenses.md` from the app bundle.
Could you please add that to [zed-editor-stable-bin.pacscript](https://github.com/pacstall/pacstall-programs/blob/master/packages/zed-editor-stable-bin/zed-editor-stable-bin.pacscript) and [zed-editor-preview-bin.pacscript](https://github.com/pacstall/pacstall-programs/blob/master/packages/zed-editor-preview-bin/zed-editor-preview-bin.pacscript). Thank you preserving the ability to install `zed-editor` and `zed-editor-preview` simultaneously.

**Comment by bibelin** [comment]
> It looks like you are failing to include `licenses.md` from the app bundle.
> Could you please add that to [zed-editor-stable-bin.pacscript](https://github.com/pacstall/pacstall-programs/blob/master/packages/zed-editor-stable-bin/zed-editor-stable-bin.pacscript) and [zed-editor-preview-bin.pacscript](https://github.com/pacstall/pacstall-programs/blob/master/packages/zed-editor-preview-bin/zed-editor-preview-bin.pacscript).

Done :)

**Comment by notpeter** [comment]
While we appreciate third party package maintainers, at this time we don't want to steer Ubuntu users towards adding an alternate package manager (pacstall) just so they can use Zed. Thanks for your enthusiasm and thanks for packaging Zed!

**Comment by bibelin** [comment]
PR closed, decision made, so I'm not gonna argue, just want to say a few more words. I understand your position, yet feel free to bring the links to pacstall packages into docs if you change your mind. :sweat_smile: 
Again, thanks for creating Zed! Best wishes! :heart:

**Pull request closed without merging**